io/www/src/fragment-client.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * Fragment Client Library
+ * A reusable client-side library for working with content fragments
+ */
+
+// Import the modules
+import { fetchFragment } from './fragment/fetch.js';
```

Comment:
I know it is a big change, but I wonder if we should write the transformers as well using es modules and convert them to common js for IO runtime deployment? We could keep leveraging web-test-runner and the usual eslint configuration.

Reply:
you mean writing them in common js, no? they are written atm in esm